README.md

```diff
@@ -1,12 +1,16 @@
 # News Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The project was about building a classical news  or magazine site. I wanted my site to feature nature and outdoors. I love the scandinavian landscapes and I choose to call the site "A fjord with no name". The posted pictures and texts refer to things to do and to find there outdoors.
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
+I started by making three layouts in figma. As this is totally new for me, it took me lots of hours to do so. I am still uncertain, if it was helpful for me or not, because I still did a lot of designing and changing the styles after I started coding.
+
+I started by desinging the mobile first and then adding responsiveness for tablet and desktop.
+
+I realised how easy it is to loose the overview in your code, even with a rather small project like mine. So, I wonder, if it would help to code section by section and make them responsive instead of designing the whole mobile look first. I will to keep an eye on that in my future projects. And I would also like to investigate about css animations.
```

Comment:
I would recommend to develop the first section of your project (ex: header) in all of three responsive designs(mobile, tablet, desktop) so you dont get lost in the process and then get to the next section.